lib/components/formik/Button.js

```diff
@@ -0,0 +1,29 @@
+import React, { forwardRef } from "react";
+import { useFormikContext } from "formik";
+import Button from "../Button";
+import { equals } from "ramda";
+
+const FormikButton = forwardRef(({ disabled, ...otherProps }, ref) => {
+  const {
+    handleSubmit,
+    isSubmitting,
+    values,
+    initialValues,
+    isValid,
+  } = useFormikContext();
+
+  const isDisabled =
+    disabled ?? (isSubmitting || !isValid || equals(values, initialValues));
```

Comment:
The value of `disable` will be `false` if not provided. We can't rely on nullish coalescing operator.
```suggestion
    disabled || (isSubmitting || !isValid || equals(values, initialValues));
```

Reply:
Unless explicitly specified, the value of `disabled` prop will be `undefined`.
We need to turn off our logic execution if it is explicitly specified as true or false by the developer. In your suggestion, the result of our logic will be considered if `disabled` is set to false, which is not the desired outcome. If the developer sets it as false, the component should not be disabled by our logic to avoid magic.